backend/uclapi/roombookings/helpers.py

```diff
@@ -151,13 +175,18 @@ def _parse_datetime(start_time, end_time, search_date):
 def _serialize_rooms(room_set):
     rooms = []
     for room in room_set:
+        if room.roomclass in ROOM_TYPE_MAP:
+            classification_str = ROOM_TYPE_MAP[room.roomclass]
+        else:
+            classification_str = "Unknown Room Type"
```

Comment:
Should this be `null`/`None`?

Reply:
I imagined this being in apps that display that value directly, but we can make it None and explain that in the docs.